🐳(PM2) install and use pm2 process manager in both images

lunika opened the pull request:
### Purpose

We wanto to use PM2 process manager in order to run learninglocker and
xapi-service images. PM2 is just installed and run without any
configuration. The configuration should be made for every projects
using these images.

### Proposal

- [x] Install `pm2` in both docker images.
- [x] Use `pm2-runtime` in docker-compose example.

sampaccoud commented on example/docker-compose.yml:
```diff
@@ -2,7 +2,7 @@ version: "3"
 services:
   mongodb:
-    image: mongo:3.2
+    image: mongo:4.0
```
should we split upgrades and adding PM2 in 2 commits?

lunika replied:
Yes we can

lunika replied:
Done

lunika committed:
🐳(dockerfile) install and use pm2 process manager in both images

We want to use PM2 process manager in order to run learninglocker and
xapi-service images. PM2 is just installed and run without any
configuration. The configuration should be made for every projects
using these images.

lunika committed:
🔧(docker-compose) upgrade mongo, learninglocker and xapi version

In order ro use pm2 process manager we want to upgrade to the latest
version available and upgrade to mongo 4 too because latest version of
learninglocker requires mongo 4

sampaccoud reviewed (approved):
Just the subject of your first commit "PM2" may be a bit narrow ;-) maybe dockerfile?
